node/amuman_node/main.py

```diff
@@ -196,3 +198,10 @@ def entrypoint() -> None:
         asyncio.run(NodeClient().start())
     except KeyboardInterrupt:
         log.warning("Caught KeyboardInterrupt (Ctrl+C). Shutting down...")
+
+
+if __name__ == "__main__":
```

Comment:
I assume this is because of debugpy, can't you just call the entrypoint function here?